examples/realtime-mc.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       [variables]
       # isolated_cores take a list of ranges; e.g. isolated_cores=2,4-7
       isolated_cores=1
+      #isolate_managed_irq=Y
```

Comment:
so this is an optional override we can use when needed ?  (to override realtime profile)

Reply:
yes, optional and can be used when needed